package.json

```diff
@@ -294,6 +299,11 @@
                     "when": "view == staticWebApps && viewItem == azureStaticEnvironment",
                     "group": "3@2"
                 },
+                {
+                    "command": "staticWebApps.openGitHubLog",
```

Comment:
I tried this out and was getting a lot of `ServerError`, I believe a 410 status code. Are you seeing that as well? Some logs hit it and others worked just fine.

Reply:
I think this is due to log files eventually getting deleted on GitHub.  I think because a lot of the builds I tested this with were made recently, I didn't run into this at all. Good catch.

Reply:
Hmm I mean mine was a brand new SWA. Maybe _too_ new? Like, I wonder if there's timing issues viewing logs while the jobs are in progress

Reply:
That sounds likely.  I'll have do some testing around that and handle it appropriately.